### PR TITLE
Translator #608

### PR DESCRIPTION
**Pull request by lnzd17**
PR for #608 
* User can now specify if they can translate English into one or more languages and they can list out those languages
* User #show page will indicate when a user has listed translation languages
* User #index page now has a translator filter tab

### CODE REVIEW

**Comment by markottaviani** [comment]
Nice!  One tweak.  From the ```/users?translator=true``` index view, how about its show the following columns:

- Name
- Known Languages
- Duties 
- Phone
- Location

That way when looking for a translator for a specific language, one doesn't have to click on everyone's profile.  

Thanks!

**Comment by lnzd17** [comment]
@markottaviani oops! Forgot that piece. Revised accordingly. 
With most recent changes in place, are all the appropriate fields for Foster Mentor still appearing?

File: app/views/users/_user_table.html.erb [hunk 1]
```diff
@@ -3,12 +3,15 @@
...
+        <% if params[:translator] %>
+        <th> Known Languages </th>
+        <% end %>
         <th>Duties</th>
         <th>Phone</th>
         <th>Location</th>
         <% if params[:foster_mentor] == 'true' %>
           <th>Mentees</th>
-        <% else %>
+        <% elsif !params[:translator] %>
```
Comment (markottaviani) [inline app/views/users/_user_table.html.erb]: ```_user.html.erb``` needs a complimentary if statement as the current_foster_dogs and coordinated_dogs values still show up.  

https://cl.ly/0H21473G2d0j

Comment (lnzd17) [inline app/views/users/_user_table.html.erb]: 👍

**Pull request merged**